internal/k8s/metrics/node/node.go

```diff
@@ -139,7 +138,7 @@ func (*reconciler) Owns() (client.Object, []builder.OwnsOption) {
 	return nil, nil
 }
 
-func (*reconciler) Watches() (*source.Kind, handler.EventHandler, []builder.WatchesOption) {
-	// return &source.Kind{Type: new(metrics.NodeMetrics)}, &handler.EnqueueRequestForObject{}
+func (*reconciler) Watches() (client.Object, handler.EventHandler, []builder.WatchesOption) {
+	// return new(metrics.NodeMetrics), &handler.EnqueueRequestForObject{}
```

Comment:
Do we need to leave this comment?

Reply:
Since this I/F change corresponds to a destructive change in the controller-runtime, I do not think it is necessary to leave a special comment as Vald regarding the change.